### PR TITLE
Updated to watch all target type fields, clear when not found

### PR DESCRIPTION
**Pull request by dmcollom**
Per Joey's request, this pull requests updates Valhalla to query Simbad2k when any of name, type, or scheme are changed in the target of an observation request.

### CODE REVIEW

File: static/js/components/target.vue [hunk 1]
```diff
@@ -185,6 +194,16 @@ export default {
         } else {
           that.lookupText = 'Could not find any matching objects';
           that.lookupFail = true;
+          for (var x in that.ns_target_params){
```
Comment (Fingel) [inline static/js/components/target.vue]: Can you explain the need for this chunk?

Comment (dmcollom) [reply static/js/components/target.vue]: It's intended to clear the fields if the user searches for something that isn't there, for the use case where a user looks for something that exists, the fields fill, and they choose to look for something else that doesn't exist. A failed lookup will clear the fields.

File: static/js/components/target.vue [hunk 2]
```diff
@@ -154,18 +155,26 @@ export default {
...
+    targetCategories() {
+      this.target.name;
+      this.target.type;
+      this.target.scheme;
+      return this.target.name + this.target.type + this.target.scheme;
     }
   },
   watch: {
-    'target.name': _.debounce(function(name){
+    'targetCategories': _.debounce(function() {
```
Comment (Fingel) [inline static/js/components/target.vue]: 'targetCategories' probably doesn't need to be a string.

File: static/js/components/target.vue (hunk 1 above)
Comment (Fingel) [reply static/js/components/target.vue]: Ok, the code is repeated here: https://github.com/LCOGT/valhalla/blob/a31cf78898f8aacccfc9c68686dd612bd69cf685/static/js/components/target.vue#L237
Any chance you can factor the "clearing" code out to keep it DRY?

Comment (dmcollom) [reply static/js/components/target.vue]: Yeah, I can likely extract it into a method so it's cleaner.

**Pull request closed without merging**